2019q1/bcm58712.md

```diff
@@ -0,0 +1,24 @@
+## Broadcom ARM64 SoC support ##
+
+Contact: Michal Stanek, <mst@semihalf.com>, Marcin Wojtas, <mw@semihalf.com>
+
+Semihalf team work continuation on a FreeBSD support for Broadcom BCM5871X SoC series.
```

Comment:
Minor edit:

The Semihalf team continued working on FreeBSD support for the Broadcom BCM5871X SoC series.

You can make it an inline link if you like
[Broadcom BCM5871X SoC series](https://...)

Reply:
Done x2